config-linux.md

```diff
@@ -183,13 +183,16 @@ If a `cgroupsPath` value is specified, the runtime MUST consistently attach to t
 Implementations of the Spec can choose to name cgroups in any manner.
 The Spec does not include naming schema for cgroups.
 The Spec does not support per-controller paths for the reasons discussed in the [cgroupv2 documentation][cgroup-v2].
-The cgroups will be created if they don't exist.
 
-You can configure a container's cgroups via the `resources` field of the Linux configuration.
-Do not specify `resources` unless limits have to be updated.
+The runtime MUST create the cgroups specified by the `cgroupsPath` if they don't exist.
+If `cgroupsPath` is empty, then the behavior is runtime implementation specific.
```

Comment:
> i think this new sentence is fine and not any more of a "loophole" than the old language

With the new language, config authors should not leave `cgroupsPath` empty unless they have read the implementation-specific docs for any runtime that might consume their config.  With the old language… I'm not sure.  It looks like this PR (as of 81c9f3116) still has the “MAY define the default” line.  So what is this new line about?  Is it just underlining that the default may not be a fixed string (e.g. maybe the runtime defaults to the container ID), so if you leave this unset you might get new cgroup or might join an existing cgroup?  If so, I'd rather clarify that *that* is what is being implementation-defined.  The current language in 81c9f3116 supports runtimes which error out on unset `cgroupsPath` (or ignore `hooks`, etc., etc.), and I don't think we want to give that much leeway here. 

  With the old language, config authors could safely leave `cgroupsPath` empty if they didn't care